### PR TITLE
Flex 2753 convert find devices fitnesse tests to cucumber

### PR DESCRIPTION
**Pull request by SamVanDijk**

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://52.16.237.165:80/OSGP/Integration-Tests?pullRequest=152) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

File: cucumber-tests-platform/src/test/java/com/alliander/osgp/platform/cucumber/steps/ws/core/configurationmanagement/SetConfigurationSteps.java [hunk 1]
```diff
@@ -129,6 +129,9 @@ public void receivingASetConfigurationRequest(final Map<String, String> requestP
         } catch (final SoapFaultClientException ex) {
             ScenarioContext.Current().put(Keys.RESPONSE, ex);
         }
+
+        final Object response = ScenarioContext.Current().get(Keys.RESPONSE);
+        System.out.println(response);
```
Comment (MartijnSipsCGI) [inline cucumber-tests-platform/src/test/java/com/alliander/osgp/platform/cucumber/steps/ws/core/configurationmanagement/SetConfigurationSteps.java]: Remove this

File: cucumber-tests-platform/src/test/resources/features/osgp-adapter-ws-core/DeviceInstallation/AddDevice.feature [hunk 2]
```diff
@@ -159,7 +159,6 @@ Feature: CoreDeviceInstallation Device Creating
     Then the add device response contains soap fault
       | FaultCode      | SOAP-ENV:Server                                                  |
       | FaultString    | UNKNOWN_ORGANISATION                                             |
-      | InnerException | com.alliander.osgp.domain.core.exceptions.UnknownEntityException |
```
Comment (MartijnSipsCGI) [inline cucumber-tests-platform/src/test/resources/features/osgp-adapter-ws-core/DeviceInstallation/AddDevice.feature]: Why is this removed

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please
